include/GUI.h

```diff
@@ -71,6 +72,8 @@ class GUI
     void setHUDInfoVisible(bool visible);
     bool getHUDInfoVisible() const;
 
+    void setCustomButton(char src[], gui::IGUIButton *button);
```

Comment:
It's better to use a reference instead of a pointer to gui::IGUIButton here. Also you better pass a string to the function as char *, preferably const.